UE4SS/src/LuaType/LuaFText.cpp

```diff
@@ -59,6 +98,18 @@ namespace RC::LuaType
             return 1;
         });
 
+        table.add_pair("Equals", [](const LuaMadeSimple::Lua& lua) -> int {
```

Comment:
I don't know if it's a good idea to have an `Equals` function for `FText`.
It exists for `FName` because it actually exists in UE as well and does something different than `operator==`/`__eq`.

I haven't looked into whether something similar to `Equals` exists in UE for `FText`, but if it does, that behavior should probably be mimicked here if possible, and if it's not possible, we shouldn't define this function until it's possible to mimic the exact behavior.

Reply:
Thanks, will look it up and either fix or remove.

Reply:
There was no "Equals" in UE sources, so removed this one. There is only an `EqualsTo` with an additional `ComparisonLevel` parameter (they explain that localized strings cannot be compared simply & directly), which is not anywhere in the headers we use, so I would rather not implement that logic here unless the users request it for something specific.